src/electron/lib/api/apis/ProjectApi.ts

```diff
@@ -60,4 +60,12 @@ export class ProjectApi implements ElectronMainApi<ProjectApi> {
     const res = await this.blix.projectManager.removeProject(this.blix, uuid);
     if (res === -1 && graphs) this.blix.graphManager.deleteGraphs(graphs);
   }
+
```

Comment:
There is no response values returned from these functions. I guess it isn't too big of a deal tho, so can probably be merged as is.